### PR TITLE
Toucan 4

### PR DESCRIPTION
**Pull request by sam-lippert**
This PR upgrades Toucan-js from 3.4.0 to 4.0.0.

### CODE REVIEW

File: .changeset/poor-rules-compare.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@hono/sentry": patch
```
Comment (yusukebe) [inline .changeset/poor-rules-compare.md:2]: `patch` should be `minor`.

File: packages/sentry/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hono/sentry",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (yusukebe) [inline packages/sentry/package.json:3]: We don't have to modify the `version` field since the changeset will update it automatically.

**Comment by yusukebe** [comment]
Hi @sam-lippert

Thank you for the PR! I've left some minor comments. Check them.

File: packages/sentry/package.json (hunk 2 above)
Comment (sam-lippert) [reply packages/sentry/package.json]: https://github.com/honojs/middleware/pull/620/commits/554a45f4dcafe84e83a9b9efb814c416835c136b

File: .changeset/poor-rules-compare.md (hunk 1 above)
Comment (sam-lippert) [reply .changeset/poor-rules-compare.md]: https://github.com/honojs/middleware/pull/620/commits/6b2ae0fbc77425efc327fb033a027f1610f8cec2

**Comment by sam-lippert** [comment]
@yusukebe Thank you for the review. I've made the requested changes.

**Comment by yusukebe** [comment]
@sam-lippert Thanks! Looks good. I'll merge it and release a new version.

**Pull request merged**